app/grandchallenge/reader_studies/views.py

```diff
@@ -404,6 +404,21 @@ def columns(self):
     def base_object(self):
         return get_object_or_404(ReaderStudy, slug=self.kwargs["slug"])
 
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+
+        self.delete_all_enabled = all(
+            not ds.answers.exists() for ds in self.get_queryset()
+        )
```

Comment:
Why do all of that work in Python? You can do this directly with `not self.base_object.answers.exists()` or `not Answer.objects.filter(reader_study=self.base_object).exists()`?

Reply:
Nice way with ORM alone! I did not think of it this way (reaching the reader study from the question).
I will get better with using those connections through object relations.